content/applications/finance/accounting/bank/reconciliation/cash_transactions.rst

```diff
@@ -0,0 +1,86 @@
+   :align: center
+   :alt: Create a new cash log.
+
+Enter the starting balance of your cash float in the :guilabel:`Starting Balance` field and save,
+but do *not* post the entry yet. Create your invoice(s) in :menuselection:`Accounting dashboard -->
+Customers --> Invoices` and select the :guilabel:`Cash` journal when registering the payment of an
+invoice.
+
+.. image:: cash_transactions/cash-payment-journal.png
+   :align: center
+   :alt: Select Cash journal upon payment registration.
+
+Head back to the **accounting dashboard**, and click again on :guilabel:`New transactions`. Click on
+the :guilabel:`Add a line` button in the :guilabel:`Transactions` tab, and enter the total of cash
+earned during the day. Add a label name according to your needs. Input the amount of
+:guilabel:`Computed Balance` in the :guilabel:`Ending Balance`, and make sure it matches your end
+cash float.
+
+Once certain the **computed balance** is correct, click :guilabel:`Save`, :guilabel:`Post`, and
+finally :guilabel:`Reconcile`. This will take you to the **bank reconciliation** page.
+
+Cash payments reconciliation
+============================
+
+On the **bank reconciliation** page, head to the **cash log** transaction you created earlier, and
+match each cash payment with your cash log. To be able to match all cash payments, make sure no
+**partner** is selected in the :guilabel:`Select Partner` field. If a **partner** is selected,
+simply click on the field and pick the blank option from the drop-down menu. When you are done,
+click on :guilabel:`Validate`.
+
+.. image:: cash_transactions/cash-payment-reconciliation.png
+   :align: center
+   :alt: Cash payments reconciliation.
+
+.. note::
+   If your ending cash balance is **over** or **under** the computed balance, you will need to add
+   another :guilabel:`Transaction` line corresponding to the **over** or **under** amount in order
+   to match it later during reconciliation.
+
+Cash reconciliation bypass
+==========================
+
+.. _bypass:
+
+To bypass the *In Payment* status, go to :menuselection:`Accounting dashboard --> Configuration -->
+Journals` and select the **cash** journal. Click on the :guilabel:`Payments Configuration` tab and
+set a `Cash` account in both fields :guilabel:`Outstanding Receipts Account` and
+:guilabel:`Outstanding Payments Account`, then save. Payments registered in the **cash** journal now
+bypass the *In Payment* status.
+
+.. image:: cash_transactions/cash-payments-config.png
+   :align: center
+   :alt: Configuration tab of the cash journal.
+
+.. tip::
+   If you use cash to pay expenses, to advance employees, or anything resembling a cash box (or if
+   you are required to keep a record of cash inflows), you can benefit from creating a second **cash
+   journal** that *does* use the **outstanding accounts**. This way, you can both generate and
+   reconcile bank statements when counting your cash float or closing your books.
```

Comment:
Not sure the beginning of the sentence is clear.
A suggestion for rephrasing : 
In case you pay expenses in cash, to pay employees in advance..... . You can simply create a second **cash journal** that *does* use the **outstanding accounts**. This way, you can both generate and reconcile bank statements when  ...
closing your fiscal year? or not sure what you say for a day ending

Reply:
I changed it up a bit, do you think it's clearer now?